### PR TITLE
Create Add lifetime.hosting domains

### PR DESCRIPTION
**Pull request by fukkani**
[Purpose of the sites referenced
We provide subdomains (SLDs) to our customers as a cost effective way to maintain a website long term for nominal cost. We use these TLDs as the basis for our client site networks.

Reason for sites to be added
Proper handling of the namespace by browsers. For cookie isolation, SSL management, Direct URL translation, and the inevitable developments in security. To prevent super cookie violation and isolate each SLD domain account from the others in the same namespace. To better isolate malicious content reports and improve our teams response to violated hosting accounts by domain. We are hopeful that this list will help us prevent all of our users from being painted with the same "Malicious code" brush in cases of accounts being violated. In short, improved security and compartmentalization.

Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [X] Description of Organization
* [X] Robust Reason for PSL Inclusion
* [X] DNS verification via dig
* [X] Run Syntax Checker (make test)

* [X] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [X] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [X] This request was _not_ submitted with the objective of working around other third-party limits

  * [X ] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [X ] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [X] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====
About the organization
https://getlifetimehosting.com : Lifetime.hosting is a budget web hosting solution providing cost effective hosting solutions to thousands of clients. Our innovation is in finding ways to reduce the end user's costs so they can spend their money on development.

About me
I am the General Manager at getlifetimehosting.com as well as several other Internet properties.

Organization Website: 

https://getlifetimehosting.com
Reason for PSL Inclusion
====

Purpose of the sites referenced
We provide subdomains (SLDs) to our customers as a cost effective way to maintain a website long term for nominal cost. We use these TLDs as the basis for our client site networks.

Reason for sites to be added
Proper handling of the namespace by browsers. For cookie isolation, SSL management, Direct URL translation, and the inevitable developments in security. To prevent super cookie violation and isolate each SLD domain account from the others in the same namespace. To better isolate malicious content reports and improve our teams response to violated hosting accounts by domain. We are hopeful that this list will help us prevent all of our users from being painted with the same "Malicious code" brush in cases of accounts being violated.
In short, improved security and compartmentalization.
Number of users this request is being made to serve:

Proposed number of users served by this request : estimate: 10000

DNS Verification via dig
=======

Results of Syntax Checker (`make test`)
=========

](https://github.com/publicsuffix/list/pull/1591)

### CODE REVIEW

**Review by dnsguru — CHANGES_REQUESTED** [review]
RE-base and try again - this arrived as a whole-cloth update to the PSL .DAT file and cannot be accepted
